test/state/precompiles.cpp

```diff
@@ -371,50 +371,37 @@ ExecutionResult ecmul_execute(const uint8_t* input, size_t input_size, uint8_t*
 ExecutionResult ecpairing_execute(const uint8_t* input, size_t input_size, uint8_t* output,
     [[maybe_unused]] size_t output_size) noexcept
 {
-    assert(output_size >= 32);
-
+    static constexpr auto OUTPUT_SIZE = 32;
     static constexpr size_t PAIR_SIZE = 192;
+    assert(output_size >= OUTPUT_SIZE);
```

Comment:
Why not just `output_size == OUTPUT_SIZE`?

Reply:
This is generic precompile's implementation framework, sometimes it passes bigger buffers, but I don't remember why currently.